src/main/kotlin/de/cyface/uploader/DefaultUploader.kt

```diff
@@ -86,21 +84,22 @@ class DefaultUploader(private val apiEndpoint: String) : Uploader {
     override fun uploadAttachment(
         jwtToken: String,
         metaData: RequestMetaData,
```

Comment:
I think to avoid uploadIdentifier from becoming `null` it will be necessary to create two kinds of `RequestMetaData`: One for attachements and one for measurements.

Reply:
See suggestion, the Identifier of the RequestMetaData is now generic, either MeasurementIdentifier or AttachmentIdentifier (extension of MeasurementIdentifier)